psl/builtin-connectors/src/postgres_datamodel_connector.rs

```diff
@@ -134,12 +134,12 @@ const SCALAR_TYPE_DEFAULTS: &[(ScalarType, PostgresType)] = &[
 
 /// Postgres-specific properties in the datasource block.
 pub struct PostgresDatasourceProperties {
-    extensions: Option<PostgresExtensions>,
+    extensions: Option<PostgresqlExtensions>,
```

Comment:
This is unrelated to the preview feature, but no objection to renaming.

Reply:
I wish we had a convention I could be pedantic about, but we sometimes us `Postgres` and sometimes `Postgresql`. In some places even the obviously bad `PostgreSql`. I think `Postgres` is the more common and my preference because it's shorter, but really, it doesn't matter at all.

Reply:
`PostgreSql` is my favorite :D

Reply:
pOsTgReSqL